setup.sh

```diff
@@ -1,21 +1,47 @@
 #!/bin/bash
 #
-# Sets up parameters for use with other scripts.  Should be run once.
+# Sets up parameters for use with other scripts.  Removes an instance of param.sh if it exists. 
 # Sample usage: bash setup.sh
-
+# Can be run for any number of times
+rm -r params.sh
 echo "First, choose the resource identifier that specifies your cluster resoure. We
 will set up this name in your ssh configuration, and use it to reference the resource (sherlock)."
 echo
 read -p "Resource identifier (default: sherlock) > "  RESOURCE
 RESOURCE=${RESOURCE:-sherlock}
 
+if [[ "${RESOURCE}" == "sherlock" ]]
+then
+   SHERLOCK=true
+else 
+   SHERLOCK=false
+fi
+
+echo "Next, please supply the domain name of the resource. If you are using farmshare or sherlock, leave as default."
+echo
+read -p "Domain Name for Resource (default: rice.stanford.edu for farmshare) > " DOMAINNAME
```

Comment:
Why can't we just automate this for the user if we already know the names to use for both sherlock and farmshare, and the user has provided that?